### PR TITLE
fix: Popconfirm loading should finish

### PR DESCRIPTION
**Pull request by zombieJ**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link

ref #40719

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     Fix Popconfirm using `Promise` to close will not exist `loading` state even when open again.       |
| 🇨🇳 Chinese |   修复 Popconfirm 使用 `Promise` 关闭时再次打开仍然是 `loading` 状态的问题。        |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Review by li-jia-nan — APPROVED** [review]

File: components/_util/ActionButton.tsx [hunk 1]
```diff
@@ -62,6 +62,7 @@ const ActionButton: React.FC<ActionButtonProps> = (props) => {
     setLoading(true);
     returnValueOfOnOk!.then(
       (...args: any[]) => {
+        setLoading(false, true);
```
Comment (yoyo837) [inline components/_util/ActionButton.tsx:65]: #40719 这个好像才被删不久

Comment (zombieJ) [reply components/_util/ActionButton.tsx:65]: 对的，出了 bug 所以换种方式实现。（其实之前 @afc163 实现过了，但是迁移到 v5 的时候 css selector 错了）

Comment (luo3house) [reply components/_util/ActionButton.tsx:65]: ActionButton 点击关闭时容器不一定真的关闭，是不是给 AB 自己也加个 destroyOnClose 好点，这个应该是内部组件吧

Comment (zombieJ) [reply components/_util/ActionButton.tsx:65]: 嗯，两个是独立的状态。之前那个 PR 合并的时候以为就一个地方用，就当耦合的删了就删了。其实不行。

**Comment by SilentFlute** [comment]
刚使用的时候发现了这个问题, 还说提个`issue`, 没想到`pr`都有了, 而且已经`merge`掉了, 这速度可以啊, 好顶赞

**Pull request merged**